karoo_gp/base_class.py

```diff
@@ -2034,31 +2028,29 @@ def fx_fitness_gene_pool(self):
                 # check if Tree meets the requirements
                 if (len(self.population_a[tree_id][3])-1 >=
                     self.tree_depth_min and self.algo_sym != 1):
-                    if self.display == 'i':
-                        print('\t\033[36m Tree', tree_id, 'has >=',
-                              self.tree_depth_min,
-                              'nodes and is added to the gene pool\033[0;0m')
+                    self.log(f'\t\033[36m Tree {tree_id} has >= {self.tree_depth_min} '
+                             'nodes and is added to the gene pool\033[0;0m',
+                             display=['i'])
                     self.gene_pool.append(self.population_a[tree_id][0][1])
-
+                    
```

Comment:
```suggestion

```
Make sure to set your editor to strip trailing whitespace and add newlines at the end of the file.

Reply:
Done! New computer whaa